ow_plexil/src/plans/FaultHandlingPattern1.plp

```diff
@@ -0,0 +1,41 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Pan in 15 degree increments as long as no antenna fault is present
+// If an antenna fault is present, wait for resolution
+// No timeout implemented
+
+#include "plan-interface.h"
+
+LibraryAction InitializeLander (InOut Boolean initialize);
+
+FaultHandlingPattern1:
+{
+  Boolean Initialize = true;
+  Real NewAngle = 0;
+  Real NumPreviousRevolutions = 0;
+
+  LibraryCall InitializeLander (initialize = Initialize);
+
+  Pattern1Image:
+  {
+    Repeat true;
+    Start !Lookup(AntennaFault) && !Initialize;
```

Comment:
The Initialize variable seems unneeded.  Initialization can be assumed by completion of InitializeLander, which by definition of a CheckedSequence (which is what this is) would have successfully completed before this node.

Reply:
I added this after initial testing because InitializeLander and Pattern1Image are under a concurrence node, and without it panning actions attempt to start concurrently so one gets skipped. Am I missing something else?